### ISSUE TITLE
[bug]: "Cancel batch" in Queue view unavailable/greyed out

### ISSUE THREAD

**Issue by mcbexx** [invoke-ai/InvokeAI#6110]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

RTX 2070S 

### GPU VRAM

8GB

### Version number

4.0.1

### Browser

Firefox 124.0.1

### Python dependencies

{
  "accelerate": "0.28.0",
  "compel": "2.0.2",
  "cuda": "12.1",
  "diffusers": "0.27.2",
  "numpy": "1.26.4",
  "opencv": "4.9.0.80",
  "onnx": "1.15.0",
  "pillow": "10.3.0",
  "python": "3.10.6",
  "torch": "2.2.1+cu121",
  "torchvision": "0.17.1+cu121",
  "transformers": "4.39.1",
  "xformers": "0.0.25"
}

### What happened

In the Queue, the option to "Cancel Batch" is greyed out for me. I can cancel individual items, but the option to cancel the corresponding batch, while visible, is no longer accessible.
This is after an update from the latest 3.7x version to 4.0.1 by following the update instructions (downloaded zip, install in existing folder).

### What you expected to happen

I expected the "Cancel Batch" button to be accessible 😀

### How to reproduce the problem

Start a queue with a batch of multiple generations, switch to queue view, click on a pending job to expand its view, "cancel batch" button is inaccessible/greyed out.

**Comment by psychedelicious** [comment]
Hmm, I can't reproduce this. Does this happen every time you queue up multiple generations? If it's not every time, is there any pattern to trigger the issue?

**Comment by mcbexx** [comment]
Happens every time as far as I can tell. Tested in Firefox (default browser on my system) and Chrome.
Tried some different settings (both SD 1.5 and SDXL checkpoints), re-used old prompts/settings, started a prompt from scratch, - I can't see any potential triggers on my end so far. I changed some settings (enable/disabled progress images, reset the web ui, cleared intermediates), This happens for queued items from txt2img, img2img and inpainting in the unified canvas alike.

Button is simply greyed out and the cursor also changes when hovering over it.
![firefox_Se6oC10N4n](https://github.com/invoke-ai/InvokeAI/assets/56156474/ac25600e-d71d-4559-b3e0-ea2c2bd45e2d)
